src/main/java/nextstep/courses/domain/SessionInfo.java

```diff
@@ -0,0 +1,33 @@
+package nextstep.courses.domain;
+
+public class SessionInfo {
+    private final SessionType sessionType;
+    private final Long price;
+    private final Integer maxStudents;
+
+    public SessionInfo(SessionType sessionType, Long price, Integer maxStudents) {
+        this.sessionType = sessionType;
+        this.price = price;
+        this.maxStudents = maxStudents;
+    }
+
+    public void checkPaymentEqualsPrice(long payment) {
+        if (sessionType == SessionType.PAID && price != payment) {
```

Comment:
객체에게 메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (sessionType.isPaid() && price != payment) {
```